src/deepak/assignment39/Addition.java

```diff
@@ -0,0 +1,32 @@
+package deepak.assignment39;
+
+public class Addition {
+
+
+	void addInteger(int a, int b) {
```

Comment:
this is not overload example @DeepakLOct2020 
Overloading: method name must be same, method parameter sequence and data type must be different.